dist/nginx.conf

```diff
@@ -56,58 +56,104 @@ server {
     try_files $uri @proxy;
   }
 
-  location ~ ^/(emoji|packs|system/accounts/avatars|system/media_attachments/files) {
-    add_header Cache-Control "public, max-age=31536000, immutable";
+  # If Docker is used for deployment and Rails serves static files,
+  # then needed must replace line `try_files $uri =404;` with `try_files $uri @proxy;`.
+  location = sw.js {
+    add_header Cache-Control "public, max-age=604800, must-revalidate";
     add_header Strict-Transport-Security "max-age=63072000; includeSubDomains";
-    try_files $uri @proxy;
+    try_files $uri =404;
   }
 
-  location /sw.js {
-    add_header Cache-Control "public, max-age=0";
+  location ~ ^/assets/) {
```

Comment:
pcre_compile() failed: unmatched parentheses in "^/assets/

Reply:
Are you checking with nginx? Which version?

Reply:
There is an unmatched `)`. Is that not a mistake?

Reply:
Ups...

Reply:
Fixed!